src/protocol/message/mod.rs

```diff
@@ -16,13 +16,15 @@ use crate::protocol::{
     },
 };
 
+pub const MAGIC_LEN: usize = 4;
```

Comment:
apologies for not having noticed it before - I think this should go to the `constants.rs` file

Reply:
Done